src/file-chooser.c

```diff
@@ -603,11 +603,115 @@ handle_save_file (XdpFileChooser *object,
   return TRUE;
 }
 
+static XdpOptionKey save_files_options[] = {
+  { "accept_label", G_VARIANT_TYPE_STRING, NULL },
+  { "modal", G_VARIANT_TYPE_BOOLEAN, NULL },
+  { "current_name", G_VARIANT_TYPE_STRING, NULL },
+  { "current_folder", G_VARIANT_TYPE_BYTESTRING, NULL },
+  { "files", G_VARIANT_TYPE_BYTESTRING, NULL },
```

Comment:
This needs to be G_VARIANT_TYPE_BYTESTRING_ARRAY

Reply:
Fixed. Thanks!